provider/clusterrosa/hcp/resource.go

```diff
@@ -351,6 +353,17 @@ func (r *ClusterRosaHcpResource) Schema(ctx context.Context, req resource.Schema
 					objectplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"ec2_metadata_http_tokens": schema.StringAttribute{
+				Description: "This value determines which EC2 Instance Metadata Service mode to use for EC2 instances in the cluster." +
+					"This can be set as `optional` (IMDS v1 or v2) or `required` (IMDSv2 only)." + common.ValueCannotBeChangedStringDescription,
+				Optional: true,
```

Comment:
Should we enforce it instead of having the optional and computed behavior?

Reply:
@gdbranco 
do you mean to change this to "Required: true"? We should not make this change IMO.
BTW, in classic cluster resource this attribute is also set as optional and computed.

Reply:
Yeah, we have been trying to move away from optional and computed values. But it's alright. I suppose it's easier anyway. As I expect the API to start returning the value for this for clusters that were not creating setting it